### PR TITLE
Update hardware-inventory.md

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

**Comment by benhylau** [comment]
Oops, please squash and merge

File: admin/hardware-inventory.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 | Item | Units | Status |
 |:-----|:------|:-------|
-| Raspberry Pi 3B+ with power | 6 | With @benhylau |
-| Toplinkst TOP-S5 | 45 | Waiting for USB adapters |
-| Samsung Evo Select 32 GB | 20 | With @benhylau |
+| Raspberry Pi 3B+ with power | 6 | With @darkdrgn2k |
+| Toplinkst TOP-S5 | 45 | With @darkdrgn2k |
```
Comment (dasanchez) [inline admin/hardware-inventory.md]: This doesn't mean that @darkdrgn2k has 45 TOP-S5s on hand, correct?
Now that @makeworld-the-better-one and @benhylau each got one of of these, I expected this number to go down by two, but I'm probably not reading this right.

**Comment by makeworld-the-better-one** [comment]
I mean technically at the moment @darkdrgn2k has mine and is soldering it. Idk if that changes anything.

**Comment by darkdrgn2k** [comment]
> I mean technically at the moment @darkdrgn2k has mine and is soldering it. Idk if that changes anything.

I think its a math problem error

> | Toplinkst TOP-S5 | 45 | Waiting for USB adapters |

If @benhylau has 45 dongles and 

>| TOP-S5 | 1 | 0.00 | @benhylau | @makeworld-the-better-one |
>| TOP-S5 | 1 | 0.00 | @benhylau | @benhylau |

And he gives 2 away 

How many dongles does he have left (to give to @darkdrgn2k)

It should be `43` not `45`

However looking at the original inventory
Another question is

If we sent 3 + 2 + 3  abroad already before this exchange
That's 8 dongles
There where 50 initially 
50-8  = 42 not 45 

So it SHOULD be 40 now with the @makeworld-the-better-one  and @benhylau  allocation?

@benhylau  did you count the 3 that you "reimbursed" me from the ones I sent to Florida. That may be the missing 3.

File: admin/hardware-inventory.md (hunk 1 above)
Comment (benhylau) [inline admin/hardware-inventory.md]:
```suggestion
| Toplinkst TOP-S5 | 40 | With @darkdrgn2k |
```

Comment (benhylau) [reply admin/hardware-inventory.md]: Sorry I forgot to subtract this. It's `50-(3+2+3+1+1) = 40`, every single one is accounted for in the _Hardware allocation_ table.

Thanks for noticing this @dasanchez

**Comment by benhylau** [comment]
@dasanchez @darkdrgn2k see https://github.com/tomeshnet/p2p-internet-workshop/pull/89#discussion_r255314983

**Pull request merged**